docs/content/integrations/airflow/from-airflow-to-dagster.mdx

```diff
@@ -26,7 +26,7 @@ with DAG(
         "retries": 1,
     },
     description="A simple tutorial DAG",
-    schedule=timedelta(days=1),
+    schedule_interval=timedelta(days=1),
```

Comment:
@Ramshackle-Jamathon want to confirm that this works across all the airflow versions we'd expect people to use this guide for?

Reply:
Pretty sure this also isn't under test (the test would have been failing)

Reply:
👍 yep i had to fix this in the SDA pr as well (not under test either)